countries/deu/deu.json

```diff
@@ -0,0 +1,26 @@
+{
+  "resources": [
+    {
+      "name": "Tagesaktuelle Informationen zum Coronavirus",
+      "link": "https://www.bundesgesundheitsministerium.de/coronavirus.html",
+      "organization": "Bundesministerium für Gesundheit"
+    },
+    {
+      "name": "COVID-19: Tipps für Eltern (PDF)",
+      "link": "https://www.bundesgesundheitsministerium.de/fileadmin/Dateien/3_Downloads/C/Coronavirus/COVID-19_Tipps_fuer_Eltern.pdf",
+      "organization": "Bundesamt für Bevölkerungsschutz und Katastrophenhilfe"
+    },
+    {
+        "name": "Tipps bei häuslicher Quarantäne",
+        "link": "https://www.bbk.bund.de/SharedDocs/Downloads/BBK/DE/Publikationen/Broschueren_Flyer/Tipps_%20haeusliche_Quarantaene.pdf?__blob=publicationFile",
+        "organization": "Bundesamt für Bevölkerungsschutz und Katastrophenhilfe"
+    },
+    {
+        "name": "Informationen zum neuartigen Coronavirus / Covid-19",
+        "link": "https://www.infektionsschutz.de/coronavirus/",
+        "organization": "Bundeszentrale für Gesundheitliche Aufklärung"
+    }
+
+  ],
+  "twitterFeed": "@BMG_Bund"
```

Comment:
I think the `twitterFeed` value needs to be without the `@` symbol :)

Reply:
Oh right, fixed!